index.html

```diff
@@ -1,41 +1,47 @@
+    <title>Triphouse</title>
+    <link rel="stylesheet" href="./styles/styles.css" />
   </head>
   <body>
-    <blockquote cite="https://www.interface.ru/home.asp?artId=25273">
-        Измерять продуктивность программиста подсчетом строк кода — это так же, как оценивать постройку самолета по его весу.
-        <strong>Bill Gates</strong>.
-    </blockquote>
-    <hr>
-    <h2> Для того, чтобы стать фронт энд-разработчиком нужно выучить:</h2>
-
-         <h3><em>Html и Css</em></h3>
-        Именно эти языки отвечают за то, как сайт будет выглядеть в браузере. Выучить их можно <del>самостоятельно с помощью интернет-ресурсов</del>. <mark>Главное</mark> - сразу применять на практике, создавая подобия сайтов. Самое главное на этом этапе - не выучить все наизусть, а получить представление о языках, их работе, атрибутах и возможностях. Когда вы будете понимать, что вам конкретно нужно, вы сможете быстро загуглить нужный тег, важнее понимать структуру в целом.
-        <h3><em>Верстка макетов</em></h3>
-        Вы можете найти гайды по верстке в интернете и шаг за шагом учиться превращать PSD-макеты в веб-страницы.
-        На этом этапе вы уже можете пробоваться на должность HTML-верстальщик.
-        <h3><em>JavaScript</em></h3>
-        Это язык программирования, который позволяет вам создать динамически обновляемый контент, управляет мультимедиа, анимирует изображения.
-
-      <h2>Советы:</h2>
-    <p>
-        1 Копайте глубже <br>
-      Каждый браузер по-разному отображает один и тот же код, но для эффективной работы нужно, чтобы ваш код был кроссбраузерным и адекватно отображался для всех пользователей. Для этого потратьте немного больше времени и разберитесь, в каком из браузеров код работает правильно и поправьте его, для адекватного отображения в остальных. Таким образом, вы сделаете правильные выводы и сэкономите время в будущем. Это касается и ситуаций, если вы слепо добавляете что-то в код до тех пор, пока не получите нужный результат. Разберитесь, чтобы не наступать на грабли каждый раз.
-      </p>
-      <p>
-          2 Читайте код <br>
-          Читая код, написанный другими разработчиками, вы улучшаете свои навыки, видите, что у задач бывает не только один вид решения. Также вы можете принять участие в открытых проектах, код которых дополняют и правят разные разработчики. Это улучшает ваши навыки, добавляет способности критически смотреть на код.
-      </p>
-
-      <p> 3 Работайте в команде <br>
-      Работа на фрилансе - это мечта всех разработчиков, однако работая в команде, вы имеете отличную возможность перенимать опыт старших девелоперов, учиться, советоваться и совершенствоваться.
-      </p>
-<p>
-    <a href="about/index.html">Об авторе</a>
-</p>
+    <div class="container">
+      <div class="top-section">
+        <div class="block-1">
+          <div class="header block-1">
+            <div class="header logo-img"></div>
+            <div class="header menu">
+              <p class="header item-1">Stays</p>
```

Comment:
It doesn't match your styles to write header here and after.

Reply:
This header-name  is from Figma names-section, as you recommended

Reply:
Can you show how should I use it?

Reply:
You styled header as flex-container. But these paragraphs doesn't have flex-elements inside. So, it doesn't make sense.